.github/workflows/run_periodic_tests.yml

```diff
@@ -86,41 +86,35 @@ jobs:
 
       - name: Install standard python dependencies
         run: |
-          python -m pip install --upgrade pip wheel setuptools
-          python -m pip install "tox<4"
-
+          python -m pip install --upgrade pip wheel setuptools nox
+  
       - name: Install SuiteSparse and Sundials
         if: matrix.os == 'ubuntu-latest'
-        run: tox -e pybamm-requires
+        run: nox -s pybamm-requires
 
-      - name: Run unit tests for GNU/Linux with Python 3.8, 3.9, and 3.10
-        if: matrix.os == 'ubuntu-latest' && matrix.python-version != 3.11
-        run: python -m tox -e unit
+      - name: Run unit tests for GNU/Linux with Python 3.8, 3.10, and 3.11
+        if: matrix.os == 'ubuntu-latest' && matrix.python-version != 3.9
+        run: nox -s unit
```

Comment:
```suggestion
      - name: Run unit tests for GNU/Linux with Python 3.8, 3.9, and 3.10
        if: matrix.os == 'ubuntu-latest' && matrix.python-version != 3.11
        run: nox -s unit
```

Reply:
Note: this will change `git blame`. It would be better to fix the merge instead of manually committing the upstream changes.